### DISCUSSION TITLE
not a question, but in 1.21~ where was theMinecraft etc relocated

### DISCUSSION THREAD

**Discussion by Smellon69** [zlataovce/takenaka#52] [General]
all of these things e.g:
theMinecraft
thePlayer
playerController
theWorld

etc are no longer where they used to be.

**Comment by zlataovce** [comment]
mappings change based on Mojang's changes, so that's expected

I can't point you to equivalent members as I don't know where those were in the first place

**Comment by Smellon69** [comment]
> mappings change based on Mojang's changes, so that's expected
> 
> I can't point you to equivalent members as I don't know where those were in the first place

they were located at net/minecraft/client/Minecraft

**Comment by zlataovce** [comment]
and in what version?

**Reply by Smellon69** [reply]
many, just not 1.21. as early as 1.8.9 to 1.16.5 and im sure many more versions.
